File: dart_exam/lib/model/customer.dart
class Customer{
  int? Id;
  String? FullName;
  DateTime? Birthday;
  String? Address;
  String? PhoneNumber;
  Customer({this.Id, this.FullName, this.Birthday, this.Address, this.PhoneNumber});
  factory Customer.valueFromJson(Map<String,dynamic> json){
    return Customer(
        id: json["Id"] as int,
        fullName: json["FullName"] as String,
        birthday: json["Birthday"] as DateTime,
        address: json["Address"] as String,
        phoneNumber: json["PhoneNumber"] as String
    );
  }
  Map<String, dynamic> toJson() {
    return {
      "Id": Id,
      "Name": FullName,
      "Birthday": Birthday,
      "Address": Address,
      "PhoneNumber": PhoneNumber,
    };
  }
  @override
  String toString(){
    return "id: $id, name:$fullName, birthday:$birthday, address:$address, phoneNumber: $phoneNumber";
  }
}

File: src/main/java/com/example/dart_exam/entity/Customer.java
package com.example.dart_exam.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@Entity
@Table (name = "customer")
@NoArgsConstructor
@AllArgsConstructor
public class Customer {
    @Id
    @GeneratedValue (strategy = GenerationType.IDENTITY)
    @Column(name = "Id")
    private Long Id;
    @Column (name = "FullName")
    private String FullName;
    @Column (name = "Birthday")
    private Date Birthday;
    @Column (name = "Address")
    private String Address;
    @Column (name = "PhoneNumber")
    private String PhoneNumber;

}
